.ci/ci.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+# Check style
+
+./.ci/check-style
+make -C py
+./.ci/check-pep8
+
+# Bootloader variants
+make -j8 bootloader
```

Comment:
Do we need `make clean` in between every call to make?

Reply:
Probably, but I want to avoid CI time to go up that much :/ Do you think we can get away like it is now?

It should work fine once those are proper cmkae targets, right?